worlds/dark_souls_3/docs/locations_en.md

```diff
@@ -0,0 +1,2269 @@
+* **PW2:** [Painted World of Ariandel (After Contraption)](#painted-world-of-ariandel-%28after-contraption%29)
+* **DH:** [Dreg Heap](#dreg-heap)
+* **RC:** [Ringed City](#ringed-city)
+
+General notes:
+
+* "Lizard" always refers to a small crystal lizard.
+
+* "Miniboss" are large enemies that don't respawn after being killed and usually
+  drop some sort of treasure, such as Boreal Outrider Knights and Ravenous
+  Crystal Lizards. They usually don't have boss bars, but a few of them do.
+
+* NPC quest items are always in the first location you can get them _without_
+  killing the NPC or ending the quest early.
+
+### High Wall of Lothric
+
+* **Back tower:** The tower _behind_ the High Wall of Lothric bonfire, past the
+  path to the shortcut elevator.
+
+* **Corpse tower:** The first tower after the High Wall of Lothric bonfire, with
+  a dead Wyvern on top of it.
+
+* **Fire tower:** The second tower after the High Wall of Lothric bonfire, where
+  a living Wyvern lands and breathes fire at you.
+
+* **Flame plaza:** The open area with many items where the Wyvern breathes fire.
+
+* **Wall tower:** The third tower after the High Wall of Lothric bonfire, with
+  the Tower on the Wall bonfire.
+
+* **Fort:** The large building after the Tower on the Wall bonfire, with the
+  transforming hollow on top.
+
+  * "Entry": The first room you enter after descending the ladder from the roof.
+
+  * "Walkway": The top floor of the tall room, with a path around the edge
+    hidden by a large wheel.
+
+  * "Mezzanine": The middle floor of the tall room, with a chest.
+
+  * "Ground": The bottom floor of the tall room, with an anvil and many mobs.
+
+* **Fountain:** The large fountain with many dead knights around it, where the
+  Winged Knight patrols in vanilla.
+
+* **Shortcut:** The unlockable path between the promenade and the High Wall of
+  Lothric bonfire, including both the elevator and the area at its base.
+
+* **Promenade:** The long, wide path between the two boss arenas.
+
+### Undead Settlement
+
+* **Foot:** The area where you first appear, around the Foot of the High Wall
+  bonfire.
+
+* **Burning tree:** The tree near the beginning of the region, with the
+  Cathedral Evangelist in front of it in vanilla.
+
+* **Hanging corpse room:** The dark room to the left of the burning tree with
+  many hanging corpses inside, on the way to the Dilapidated Bridge bonfire.
+
+* **Back alley:** The path between buildings leading to the Dilapidated Bridge
+  bonfire.
+
+* **Stable:** The building complex across the bridge to the right of the burning
+  tree.
+
+* **White tree:** The birch tree by the Dilapidated Bridge bonfire, where the
+  giant shoots arrows.
+
+* **Sewer:** The underground passage between the 
+
+* **Chasm:** The chasm underneath the bridge on the way to the tower. It's
+  possible to get into the chasm without a key by dropping down next to Eygon of
+  Carim with a full health bar.
```

Comment:
If the plan is to have it require Grave Key, then I would recommend that the logic require Grave Key, and have the drop be a trick that you could do outside of logic. 

If you want to keep the trick in logic, then I think that the note about it here is fine. Otherwise, I don't think it's necessary. I don't have a strong feeling either way, so you can mark this conversation as resolved when you've made a final decision.